src/Draco.Formatter.CSharp.Tests/FormatterTest.cs

```diff
@@ -0,0 +1,36 @@
+using Draco.Formatter.Csharp;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+using Xunit;
+using Xunit.Abstractions;
+namespace Draco.Formatter.CSharp.Test;
+public sealed class FormatterTest(ITestOutputHelper logger)
+{
+    [Fact]
+    public void SomeCodeSampleShouldBeFormattedCorrectly()
+    {
+        var input = """"
+            class Program
+            {
+                public static void Main()
+                {
+                    Console.WriteLine("Hello, World!");
+                }
+            }
+
+            """";
+        var tree = SyntaxFactory.ParseSyntaxTree(SourceText.From(input));
+        var formatted = CSharpFormatter.Format(tree);
+        logger.WriteLine(formatted);
+        Assert.Equal(input, formatted, ignoreLineEndingDifferences: true);
+    }
+    [Fact]
+    public void ThisFileShouldBeFormattedCorrectly()
```

Comment:
```suggestion
    }
    
    [Fact]
    public void ThisFileShouldBeFormattedCorrectly()
```

Reply:
This test case validate against this file, since the formatter lake features, that's why you see it's cramped like that.